content/news/2023.md

```diff
@@ -0,0 +1,49 @@
+# 2023
+
+*You'll find below all the updates of our SaaS offers. Enjoy!*
+
```

Comment:
## January 2023

### <span class="label label-news label-updated">UPDATED</span> GUIDED TUTORIALS - Copy button and new theme for code snippet
-   Easily copy/past any code snippet from our guided tutorials now with our new copy button!  
-   A new theme for our code snippet has been implemented, [tell us what you think about it](https://api.akeneo.com/tutorials/how-to-get-your-app-token.html) to have it deployed in all the technical documentation 🚀
👉 [Start building your app now ](https://api.akeneo.com/tutorials/how-to-get-your-app-token.html)